### PR TITLE
Added BigQuery Getting Started Documentation

### PR DESCRIPTION
**Pull request by paulcalvano**
This is an update to the documentation on the legacy HTTP Archive repo - https://github.com/HTTPArchive/legacy.httparchive.org/blob/master/docs/bigquery-gettingstarted.md

https://github.com/HTTPArchive/httparchive.org/issues/52

### CODE REVIEW

**Comment by rviscomi** [comment]
Thanks for migrating this @paulcalvano! A couple of notes:

- after giving the link to add the HA project to BQ Beta, can you also suggest pinning the project? otherwise I'm not sure if it will be there if the user doesn't follow that special link again.
- found a broken image ("SQL Dialect"):

image

**Comment by paulcalvano** [comment]
Updated the image.   Testing this with a new account, and pinning the  project doesn't work in the new UI.   I'll add instructions for pinning the project via the classic UI and we can update this once the beta UI's functionality is updated.

**Comment by paulcalvano** [comment]
Done with my updates. Let me know if this looks ok.

File: docs/gettingstarted_bigquery.md [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+## Useful Links for BigQuery SQL Reference
+BigQuery supports two SQL dialects: standard SQL and legacy SQL.  Legacy SQL is a non-standard SQL dialect that BigQuery started out using.   Standard SQL is a a SQL2011 compliant dialect that has many more features and functions.  Standard SQL is the preferred dialect for querying data via BigQuery, but both are supported.
+
+[Documentation for Standard SQL](https://cloud.google.com/bigquery/docs/reference/standard-sql/)
+
+[Documentation for Legacy SQL](https://cloud.google.com/bigquery/docs/reference/legacy-sql)
+
+If you have existing Legacy SQL that you are trying to migrate to Standard SQL, then you may want to read the [migration guide.](https://cloud.google.com/bigquery/docs/reference/standard-sql/migrating-from-legacy-sql)
+
+When you are ready to start writing queries, make sure that the SQL dialect option selected matches what you are writing your query in.   The classig UI defaults to Legacy SQL, but de-selecting this will switch to Standard SQL.  The new UI defaults to Standard SQL.
```
Comment (rviscomi) [inline docs/gettingstarted_bigquery.md]: s/classig/classic

**Comment by rviscomi** [comment]
LGTM! Just one typo that needs to be fixed.

File: docs/gettingstarted_bigquery.md (hunk 1 above)
Comment (paulcalvano) [reply docs/gettingstarted_bigquery.md]: fixed, thanks!

**Pull request merged**